### PR TITLE
Remove dependency on jvm-verifier repository

### PR DESCRIPTION
**Pull request by brianhuffman**
This removes some of the few remaining dependencies on the
jvm-verifier repository.

### CODE REVIEW

**Comment by RyanGlScott** [comment]
Hooray!

There are a number of lingering referenced to `jvm-verifier` and `jss` that are now obsolete:

https://github.com/GaloisInc/saw-script/blob/14679870c91c53cf6f4470143cee34b7a8952933/.github/ci.sh#L129

https://github.com/GaloisInc/saw-script/blob/14679870c91c53cf6f4470143cee34b7a8952933/.gitmodules#L16-L18

https://github.com/GaloisInc/saw-script/blob/14679870c91c53cf6f4470143cee34b7a8952933/README.md#related-packages

https://github.com/GaloisInc/saw-script/blob/14679870c91c53cf6f4470143cee34b7a8952933/intTests/test0018_jss_class_load/README

https://github.com/GaloisInc/saw-script/blob/14679870c91c53cf6f4470143cee34b7a8952933/intTests/test0018_jss_class_load/test.sh#L9

https://github.com/GaloisInc/saw-script/blob/14679870c91c53cf6f4470143cee34b7a8952933/src/SAWScript/Crucible/JVM/BuiltinsJVM.hs#L82

https://github.com/GaloisInc/saw-script/blob/14679870c91c53cf6f4470143cee34b7a8952933/intTests/runtests.sh#L58

**Comment by brianhuffman** [comment]
It looks like the CI tests are working so far, so I think this is ready for review.

File: intTests/support/find-java-rt-jar.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+# Find the Java core classes JAR, 'rt.jar' (called 'classes.jar' on OS
```
Comment (RyanGlScott) [inline intTests/support/find-java-rt-jar.sh:3]: This is a bit unrelated to the spirit of the PR itself, but it feels a bit unnecessary to manually search for `rt.jar` now that `saw` supports finding `rt.jar`'s location by way of `--java-bin-dirs`/`PATH`. Perhaps we should open an issue as a reminder to see if we can avoid having to use this script.

File: intTests/test0019_jss_switch_statement/test.sh [hunk 2]
```diff
@@ -1,7 +1,3 @@
 set -e
 
-$JSS Test1
```
Comment (RyanGlScott) [inline intTests/test0019_jss_switch_statement/test.sh:3]: Now that all of the `jss`-related bits are gone, it doesn't make much sense to refer to this subdirectory with the name `test0019_jss_switch_statement`. Perhaps we should rename it.

Comment (brianhuffman) [reply intTests/test0019_jss_switch_statement/test.sh:3]: Yeah, I was wondering about that too. I completely removed the other `jss` tests, but this one seemed to have a non-`jss` part with some non-trivial content, so I thought it should remain. We should definitely change the name; perhaps `test0019_jvm_switch_statement` would do nicely.

There are also a few mentions of JSS in comments and READMEs of some of the other integration tests; I should probably edit those as well.

**Review by atomb — APPROVED** [review]
🎉

File: intTests/support/find-java-rt-jar.sh (hunk 1 above)
Comment (RyanGlScott) [reply intTests/support/find-java-rt-jar.sh:3]: I've opened #1061 for this.

**Pull request merged**